indexing-hadoop/src/main/java/io/druid/indexer/HadoopTuningConfig.java

```diff
@@ -191,6 +196,11 @@ public boolean getUseCombiner()
     return useCombiner;
   }
 
+  @JsonProperty
+  public Boolean getBuildV9Directly() {
```

Comment:
since buildV9Directly is never null, this should probably be `boolean` instead of `Boolean`